portfolio/src/main/webapp/script.js

```diff
@@ -141,3 +142,40 @@ async function displayNickname() {
     element.innerHTML = 'Your current nickname is ' + nickname + '.';
   }
 }
+
+/**
+ * Creates a map of my favorite places in Orange County
+ */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 33.6, lng: -117.77},
+    zoom: 10
+  });
+
+  addLandmark(map, 33.649397, -117.742604, 'Irvine Spectrum Center', 'places/spectrum.html');
+  addLandmark(map, 33.574302, -117.840324, 'Crystal Cove State Park', 'places/crystalcove.html');
+  addLandmark(map, 33.476702, -117.720383, 'Salt Creek Beach', 'places/saltcreek.html');
+  addLandmark(map, 33.558982, -117.668982, 'The Shops at Mission Viejo', 'places/mvmall.html');
+  addLandmark(map, 33.576321, -117.726727, 'Regal Edwards Aliso Viejo & IMAX', 'places/movies.html');
+  addLandmark(map, 33.690977, -117.888958, 'South Coast Plaza', 'places/scplaza.html');
+  addLandmark(map, 33.812095, -117.918980, 'Disneyland Park', 'places/disney.html');
+  addLandmark(map, 33.637634, -117.592207, 'Mathnasium of RSM', 'places/mathnasium.html');
+  
+}
+
+/**
+ * Adds a marker that shows an info window when clicked.
+ */
+function addLandmark(map, lat, lng, title, URL) {
```

Comment:
nit, I think variables like this should just be lowercase.  `URL` with all caps is actually a JavaScript interface, so by using that same variable name you are hiding the globally accessible `URL` (https://developer.mozilla.org/en-US/docs/Web/API/URL)

Reply:
Got it. I didn't know about the JavaScript interface either.